client/layout/activity-panel/panels/reviews.js

```diff
@@ -2,101 +2,167 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 import { Component, Fragment } from '@wordpress/element';
 import Gridicon from 'gridicons';
+import interpolateComponents from 'interpolate-components';
+import { noop } from 'lodash';
 
 /**
  * Internal dependencies
  */
+import { ActivityCard, ActivityCardPlaceholder } from '../activity-card';
 import ActivityHeader from '../activity-header';
+import Gravatar from 'components/gravatar';
+import Link from 'components/link';
+import ProductImage from 'components/product-image';
+import { ReviewRating } from 'components/rating';
```

Comment:
We should probably standardize on whether a component exports itself or an object by default - maybe this is a container vs presentational component distinction, and we should split apart the components folder… 🤔 

(just pondering, not an actionable review comment)

Reply:
In this case, the reason this is different is `components/rating` offers 3 different but related components. A base `Rating` component, and then a `ReviewRating` and `ProductRating` which provide wrappers around Rating, allowing you to just pass in a review and product object respectively.